### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by d4straub**
Main changes:

- DSL2 conversion
- three types of input supported: data folder, sample sheet, ASV fasta file
- ASV generation is now performed by DADA2 in R instead of DADA2 in QIIME2
- more reference taxonomy supported out of the box (SILVA, UNITE, PR2, GTDB, RDP)
- more data types supported (Illumina SE & PE, PacBio, IonTorrent)

see the CHANGELOG for details. 

The most recent changes from the DSL2 template are not included here. This will wait until the DSL2 template is actually released.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
 - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
 - [x] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -76,11 +76,17 @@ If `-profile` is not specified, the pipeline will run locally and expect all sof
...
+* `charliecloud`
+  * A generic configuration profile to be used with [Charliecloud](https://hpc.github.io/charliecloud/)
+  * Pulls software from Docker Hub: [`nfcore/ampliseq`](https://hub.docker.com/r/nfcore/ampliseq/)
 * `conda`
-  * Please only use Conda as a last resort i.e. when it's not possible to run the pipeline with Docker, Singularity or Podman.
+  * Please only use Conda as a last resort i.e. when it's not possible to run the pipeline with Docker, Singularity, Podman, Shifter or Charliecloud.
   * A generic configuration profile to be used with [Conda](https://conda.io/docs/)
   * Pulls most software from [Bioconda](https://bioconda.github.io/)
-* `test`, `test_multi`, `test_manifest`, `test_full`, `test_pacbio_its`, `test_doubleprimers`
+* `test`, `test_multi`, `test_manifest`, `test_full`, `test_pacbio_its`, `test_iontorrent`, `test_doubleprimers`
```
Comment (skrakau) [inline docs/usage.md:89]:
```suggestion
* `test`, `test_multi`, `test_full`, `test_pacbio_its`, `test_iontorrent`, `test_doubleprimers`
```

`test_manifest` was deleted ....

**Review by skrakau — APPROVED** [review]
Lots of changes :D looks very good to me!

File: docs/usage.md (hunk 1 above)
Comment (d4straub) [reply docs/usage.md:89]: True! That was not needed any more.

**Review by erikrikarddaniel — APPROVED** [review]
Well done @d4straub! :-)

**Pull request merged**